src/components/CohortBuilder/Categories/index.js

```diff
@@ -156,6 +184,13 @@ class Categories extends React.Component {
           onSqonUpdate={this.handleSqonUpdate}
           fields={CATEGORY_FIELDS.quickSearch}
           color={theme.filterPurple}
+          setSearchField={this.handleSearchField}
+          setActiveCategory={this.setActiveCategory}
+          category={CATEGORY_NAMES.quickSearch}
+          currentCategory={currentCategory}
+          fields={CATEGORY_FIELDS.quickSearch}
+          currentSearchField={currentSearchField}
+          onClose={this.handleCategoryClose}
```

Comment:
are all these props being used by this `Category`? `setSearchField` doesn't seem to be used, also this `Category seems to be receiving a lot more than the others.

Reply:
nope, nice catch! removed dups and unused props :)